migrations/2_deploy_contracts.js

```diff
@@ -6,7 +6,9 @@ module.exports = async (deployer, network) => {
     const datacoinAddress = mainnet ? "0x0cf0ee63788a0849fe5297f3407f701e122cc023" : "0x8e3877fe5551f9c14bc9b062bbae9d84bc2f5d4e"
     const streamrUpdaterAddress = "0xb6aA9D2708475fB026a8052E20e63AeA23233613"
     const ownerAddress = mainnet ? "0x1bb7804d12fa4f70ab63d0bbe8cb0b1992694338" : "0x3F2dA479B77cB583C3462577DCD2e89B965fe987"
-    await deployer.deploy(Marketplace, datacoinAddress, streamrUpdaterAddress, { gas: 6000000 })
+    //what is the marketplace address on test net?
```

Comment:
Looks like a TODO, could be removed. @jtakalai can confirm the answer to the question, but AFAIK there isn't really a proper testnet marketplace, as there isn't a deployed app environment which would utilize it. In that case maybe replace this comment with a comment that "there isn't a previous testnet marketplace".